cpp/src/strings/convert/convert_urls.cu

```diff
@@ -19,19 +19,27 @@
 #include <cudf/detail/get_value.cuh>
 #include <cudf/detail/null_mask.hpp>
 #include <cudf/detail/nvtx/ranges.hpp>
+#include <cudf/detail/utilities/cuda.cuh>
+#include <cudf/detail/utilities/integer_utils.hpp>
 #include <cudf/strings/convert/convert_urls.hpp>
 #include <cudf/strings/detail/utilities.cuh>
 #include <cudf/strings/detail/utilities.hpp>
 #include <cudf/strings/string_view.cuh>
 #include <cudf/strings/strings_column_view.hpp>
+#include <cudf/types.hpp>
 #include <cudf/utilities/span.hpp>
 
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/device_uvector.hpp>
 
+#include <cub/cub.cuh>
+
 #include <thrust/binary_search.h>
 #include <thrust/count.h>
```

Comment:
I don't think the `thrust/binary_search.h` or the `thrust/count.h` headers are need anymore.

Reply:
Addressed in https://github.com/rapidsai/cudf/pull/8622/commits/81af59c0d2509b3779478dc7b2d4d1e843964876.